sections/units.tex

```diff
@@ -65,9 +66,10 @@ \subsection*{Unit Card Anatomy}
 If there are multiple ties on both sides (for instance two attackers and two defenders with the same initiative), alternate between the attackers and defenders starting with the attacker.\par
 \bigskip
 
-\begin{center}
-  \includegraphics[width=0.5\linewidth]{\art/frenzy.png}
-\end{center}
+\begin{scaledfigure}[blanker]
+	\centering
+	\includegraphics[width=0.5\linewidth, height=\myspace, keepaspectratio]{\art/frenzy.png}
+\end{scaledfigure}
```

Comment:
This is a small graphic, which gets moved to the other column, once page numbers are appended. Does this figure prevent it from doing so?

Reply:
Ah yeah, just check it, it does 👍🏻